spec/SMS_spec.rb

```diff
@@ -0,0 +1,28 @@
+require './docs/SMS'
+
+describe SMS do
+
+  subject(:sms) { described_class.new }
+
+  before do
+    allow(sms).to receive(:sms)
```

Comment:
Much prefer this was of stubbing the SMS method directly rather than trying to stub the client like I did